pkg/management/postgres/configuration.go

```diff
@@ -169,15 +169,15 @@ func buildLDAPConfigString(cluster *apiv1.Cluster, ldapBindPassword string) stri
 			"search attribute", ldapConfig.BindSearchAuth.SearchAttribute,
 			"search filter", ldapConfig.BindSearchAuth.SearchFilter)
 
-		ldapConfigString += fmt.Sprintf(" ldapbasedn=\"%s\" ldapbinddn=\"%s\" ldapbindpasswd=\"%s\"",
+		ldapConfigString += fmt.Sprintf(` ldapbasedn="%s" ldapbinddn="%s" ldapbindpasswd="%s"`,
 			ldapConfig.BindSearchAuth.BaseDN, ldapConfig.BindSearchAuth.BindDN,
-			strings.ReplaceAll(ldapBindPassword, "\n", "\\\n"))
+			strings.ReplaceAll(ldapBindPassword, "\n", `\n`)) // escape newlines
```

Comment:
I find this line confusing. I would have preferred to use the same quote type here, as it was before.

Reply:
you're probably right. I wondered too. Reverting.